test/set.js

```diff
@@ -196,6 +196,17 @@ describe('Sets', () => {
     expect(array).toEqual([1, 2, 3])
   })
 
+  test('reduce', () => {
+    const set = Set.of([1, 2, 3, 4, 5])
+    const emptySet = Set.of([])
+
+    expect(set.reduce((sum, value) => sum + value, 0)).toBe(15)
+    expect(set.reduce((sum, value) => sum + value, 5)).toBe(20)
+    expect(emptySet.reduce((sum, value) => sum + value, 5)).toBe(5)
+    expect(set.reduce((sum, value) => sum + value)).toThrow(TypeError)
```

Comment:
The failing test needs a refinement: when asserting `.toThrow`, you must provide a function that returns the executed code throwing the error. The function is necessary because the code will be processed before jest can assert the error. The thrown error interrupts any further execution flow and exits right away

```suggestion
    expect(() => set.reduce((sum, value) => sum + value)).toThrow(TypeError)
```

Reply:
Thank you @marcuspoehls! Didn't think of that haha